spec/features/drag_and_drop_spec.rb

```diff
@@ -31,7 +31,9 @@
 
     # change id: 1, 2, 3, 4 to 4, 1, 2, 3
     expect do
-      first_target.drag_to(last_target)
+      Redmine::VERSION::STRING < '4.2' ?
```

Comment:
redmine-4.10.x or later of redmine-4.x will never be released, I think. So, this condition is ok.